contact-widgets.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Contact Widgets
  * Description: Beautifully display social media and contact information on your website with these simple widgets.
- * Version: 1.3.3
+ * Version: 1.3.4
```

Comment:
@EvanHerman That's weird, it shouldn't show off as a change then. Okay moving on!